BuildScripts/create_wheels.sh

```diff
@@ -77,18 +77,22 @@ function main()
 	# create the wheel for python 3.6
 	build_python_wheel 3 6 1 win32 
 	build_python_wheel 3 6 1 win_amd64
+	build_python_whell 3 6 1 mingw
```

Comment:
hi thanks for contributing. There is a small typo here. would you mind correcting the function name to "wheel" from "whell". After that I should have a chance to run it locally and get a package pushed up to pypi for you.

Reply:
i'm sorry! done!